### ISSUE TITLE
Expo crashes when I try to run on android emulator (with no message)

### ISSUE THREAD

**Issue by shqnks** [expo/expo-cli#2781]
On a react-native project, since a few hours, I have a huge issue : Expo doesn't want to open my App on my android emulator.

I'm doing `expo start` or even `expo-cli start` in the terminal,

I have the terminal interface with the QR Code, and the web page.

And when I press `a` to open on android, in my terminal I can see `Opening on Android...` and nothing happens

So I've tried to use the Metro Bundler graphic interface in the web browser to open my app and when I click on the button `Run on Android device/emulator`, I can see the message `Attempting to open a simulator and nothing happens`.

I've restarted my computer many times, I've tried `npm run --clear-cache`, `expo r -c` and still, nothing happen when I want to open my App on my emulator.

4 hours of trying to solve this and I found no solutions...

I can use my Application on my iPhone using the Tunnel connexion
When I try to open the Application on an android emulator, expo crashes. I can't even close it with CTRL+C on the terminal

I even did `npm -g uninstall expo-cli --save` and `npm install -g --force expo-cli` and still nothing happens when I try to open on Android Emulator.

My `expo diagnostics` :

Expo CLI 3.27.14 environment info:
    System:
      OS: Windows 10 10.0.18362
    Binaries:
      Node: 12.18.0 - C:\Program Files\nodejs\node.EXE
      npm: 6.14.4 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  4.0.0.0 AI-193.6911.18.40.6821437
    npmPackages:
      expo: ~39.0.2 => 39.0.3
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-39.0.3.tar.gz => 0.63.2
      react-native-web: ~0.13.7 => 0.13.18
    Expo Workflow: managed

**Comment by AdamJNavarro** [comment]
Hey @shqnks, couple questions for you!

1. If you haven't already, could you try creating a brand new project and running it on the emulator so we can determine if this is project specific?

2. Has this worked before? If so, can you let me know what versions in your environment have changed (if any)?

Cheers,
Adam

**Comment by shqnks** [comment]
Hey @AdamJNavarro 
 
I solved my problem by creating a brand new Android Device Emulator.

There was something wrong with the emulator i've been using for a couple of weeks that made Expo not able to find it. I don't know what was the problem, and why Expo crashed instead of telling me that something was wrong with the emulator.

Well now everything is all right for me :)

I'll still answer your questions : 
1: I've did it many times yes
2: Yes it worked for a couple of weeks before the problem happened.

Thanks for you reply, have a nice day.
